aci/resource_aci_rtctrlattrp.go

```diff
@@ -91,7 +116,136 @@ func setRtctrlSetTagAttributes(rtctrlSetTag *models.RtctrlSetTag, d *schema.Reso
 	if err != nil {
 		return d, err
 	}
-	d.Set("set_route_tag", rtctrlSetTagMap["tag"])
+
+	set_route_tag, err := strconv.Atoi(rtctrlSetTagMap["tag"])
+	if err != nil {
+		return d, err
+	}
+
+	d.Set("set_route_tag", set_route_tag)
+	return d, nil
+}
```

Comment:
Any reason the route_tag as everything in one function when all the other have a get and set function?